Cloud Enterprise/Getting Started Examples/aws/terraform/main.tf

```diff
@@ -1,19 +1,19 @@
+terraform {
```

Comment:
If this works with TF .12, I wouldn't require any version to be enforced
TF had some well known breaking changes from .11 to .12, but that not the case usually (maybe want to just commend out 'written with terraform vX.X just for reference)